parasol/management/commands/solr_schema.py

```diff
@@ -0,0 +1,89 @@
+'''
+**solr_schema** is a custom manage command to update the configured
+schema definition for the configured Solr instance.  Reports on the
+number of fields that are added or updated, and any that are out of
+date and were removed.
+
+Example usage::
+
+    python manage.py solr_schema
+
+'''
+
+from django.core.management.base import BaseCommand, CommandError
+import requests
+
+from parasol.solr.django import SolrClient
+from parasol.schema import SolrSchema
+
+
+class Command(BaseCommand):
+    '''Configure Solr schema fields and field types.'''
```

Comment:
Ah, I haven't been paying attention to that one. `"""` feels a little more verbose to me for some reason, but if it's PEP8 then maybe we should. I don't feel strongly about this one, what do you think?

Reply:
I'd say we should take the time to shift it over, since this is a pretty new project. I nearly did to begin with and waffled on the subject. The key wording found in https://www.python.org/dev/peps/pep-0257/ is `For consistency, always use """triple double quotes""" around docstrings. Use r"""raw triple double quotes""" if you use any backslashes in your docstrings. For Unicode docstrings, use u"""Unicode triple-quoted strings""".`